Simonyi/Non-Standard-Operations/MTCS/MTHexRot/MTHex-PXI-Controller-Reboot.rst

```diff
@@ -213,16 +213,17 @@ Hard Reboot
 
     .. figure:: /Simonyi/Non-Standard-Operations/_static/mtrot-controller-pxi-reboot-1.jpeg
    
-    B.  For *MTM2Hexapod* the netbooter is required to power cycle the PXI and drives: 
-        Connect to the M2 Hex Netbooter *m2-hexapod-netbooter.cp.lsst.org* using the credential stored in the *1Password MainTel 
-        Vault* under *M2 Hexapod PXI/Drive NetBooter*.
+    B.  For *MTM2Hexapod* while in the LSST-WAP, 
+        connect to *https://pdu1-tea-as02.cp.lsst.org* using the credentials stored in the *MainTel vault* 
+        of *LSST 1Password* as *pdu1-tea-as02.cp.lsst.org*. Click on :guilabel:`Outlets` on the left hand 
+        side menu to open the outlets screen. 
 
 #.  **Power Cycle PXI and drives**: To **power** :guilabel:`Off` the system, first power off the PXI, followed by 
     the drive. 
     
-    A.  For *MTCamHexapod*, PXI is energized through outlet 8, while the drives correspond to outlet 4 in the *PDU*.
+    A.  For *MTCamHexapod*, PXI is energized through *outlet 8*, while the drives correspond to *outlet 4* in the *https://tea-pdu01.cp.lsst.org PDU*.
```

Comment:
1. *outlet 8* should be :guilabel:`Outlet 8` because its a clickable and coonsistent with previous notation. It will also highlight the OUtlet number as we do not want anyone accidently switching something else off 
2. *outlet 4* -- > :guilabel:`Outlet 4`